packages/klevu-ui/src/components/klevu-recommendations/klevu-recommendations.stories.ts

```diff
@@ -26,6 +26,6 @@ export const Recommendations: StoryObj<KlevuRecommendations> = {
     current-product-id=${ifDefined(args.currentProductId)}
     item-group-id=${ifDefined(args.itemGroupId)}
     recommendation-id=${ifDefined(args.recommendationId)}
-    recommendation-title=${ifDefined(args.recommendationTitle)}
+    recommendation-title=${ifDefined(args.recommendationTitle || undefined)}
```

Comment:
It will be undefined. No need to add `|| undefined`

Reply:
its coming as empty string. since the code checks for undefined, I had to do this. is there a way to remove the attribute in story book?